docs/guides/command_handling.mdx

```diff
@@ -0,0 +1,734 @@
+  'alphabet',
+  'Outputs the alphabet',
+  (IContext context) {
+    context.respond(MessageBuilder.content('ABCDEFGHIJKLMNOPQRSTUVWXYZ'));
+  },
+  // Since this command is spammy, we can use a cooldown to restrict it's usage:
+  checks: [
+    CooldownCheck(
+      CooldownType.user,
+      Duration(seconds: 30),
+    )
+  ],
+);
+
+commands.addCommand(alphabet);
+```
+
+## Using converters override
+
+Converter overrides allow you to override which [`Converter`] is used to convert a specific argument.
+They can be used, for example, to filter the output of a converter so that only certain
+instances are ever passed to the function.
+You might have noticed that if we run the command `!say ""`, the bot will throw an error
+because it cannot send an empty message. This is because the string [`Converter`] saw `""` and
+interpreted it as an empty string - which the bot then tried to send.
+
+To counter this, let's create a converter that only allows non-empty strings:
+```dart
+// Since the converter is going to be passed into a decorator, it must be `const`. As such, any
+// functions used must either be static methods or top-level functions.
+// You can see the implementation of `filterInput` at the bottom of this file.
+const Converter<String> nonEmptyStringConverter = CombineConverter(stringConverter, filterInput);
+
+ChatCommand betterSay = ChatCommand(
+  'better-say',
+  'A better version of the say command',
+  (
+    IContext context,
+    @UseConverter(nonEmptyStringConverter) String input,
+  ) {
+    context.respond(MessageBuilder.content(input));
+  },
+);
+
+commands.addCommand(betterSay);
+
+String? filterInput(String input, Context context) {
+  if (input.isNotEmpty) {
+    return input;
+  }
+  return null;
+}
+```
+
+At this point, if you run the file, a new command `better-say` will have been added to the bot.
+Attempting to invoke it with an empty string (`!better-say  ""`) will cause the argument to
+fail parsing.
+
+That's all! Well done, you've reached the end of this guide. You're now able to go and create your own bot on your side.
+Remember; You just need to create a function or a class in a specific file, importing it, and register the function.
+
+
+[`nyxx_commands`]: https://github.com/nyxx-discord/nyxx_commands/tree/main
+[nyxx_commands]: https://github.com/nyxx-discord/nyxx_commands/tree/main
+[`chatcommand`]: https://pub.dev/documentation/nyxx_commands/latest/nyxx_commands/ChatCommand-class.html
+[`chatgroup`]: https://pub.dev/documentation/nyxx_commands/latest/nyxx_commands/ChatGroup-class.html
+[`icontext`]: https://pub.dev/documentation/nyxx_commands/latest/nyxx_commands/IContext-class.html
+[`execute`]: https://pub.dev/documentation/nyxx_commands/latest/nyxx_commands/ChatCommand/execute.html
+[`children`]: https://pub.dev/documentation/nyxx_commands/latest/nyxx_commands/ChatGroup/children.html
+[`addcommand`]: https://pub.dev/documentation/nyxx_commands/latest/nyxx_commands/ChatGroup/addCommand.html
```

Comment:
Casing is incorrect in the member names

Reply:
I've cased correctly the first time, that's prettier that formatted in lower case, I'll fix this asap.